tests/integration/defs/verl/test_verl_E2E_standalone.py

```
@@ -0,0 +1,164 @@
+
+@pytest.mark.timeout(1800)
+def test_verl_E2E_standalone():
+    """STANDALONE async-RL + TRT-LLM rollout — per-step weight-sync guard."""
+    model_path = os.path.join(
+        os.environ["TRTLLM_TEST_MODEL_PATH_ROOT"], "Qwen/Qwen2.5-0.5B-Instruct"
+    )
+    data_dir = _ensure_gsm8k("/tmp/verl-data/gsm8k")
+    # Suffix matches ``_run_verl_train``'s naming so any out-of-band log
+    # rescue tooling that greps for ``*-verl-train.log`` (e.g. local Lyris
+    # post-mortem scripts) picks this up too.
+    log_file = tempfile.mktemp(prefix="standalone-", suffix="-verl-train.log")
```

Ruff (0.15.12)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)